packages/app/src/contacts/components/contact-name/contact-name.interface.ts

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
(not related to this file, just it is easier to have a thread)

After performing specific steps, it is not possible to go back to the previously opened thread

https://user-images.githubusercontent.com/41268731/187693091-1ca37127-2139-4a2f-9b17-54f96057952c.mov